### ISSUE TITLE
 Validation and Test

### ISSUE THREAD

**Issue by GGchen1997** [thuml/Time-Series-Library#366]
https://github.com/thuml/Time-Series-Library/blob/9a875da4b898fcc857c98e14b147ffc7fd45024b/exp/exp_classification.py#L80

Hi, I am wondering it is normal here to use test set as validation set.

**Comment by GGchen1997** [comment]
This is like using test set to choose the checkpoint.

**Comment by wuhaixu2016** [comment]
Hi, thanks for coming up with this important question.

We use the test set as validation beacuse the UEA benchmark only provides train and test subsets. This is not a rigorous setting, but it can still be used to measure the upper bound of different models.
